pkgs/development/libraries/pango/default.nix

```diff
@@ -39,10 +40,9 @@ stdenv.mkDerivation rec {
     meson ninja
     glib # for glib-mkenum
     pkg-config
-    gobject-introspection
     gi-docgen
     python3
-  ];
+  ] ++ lib.optional withIntrospection gobject-introspection;
```

Comment:
In GNOME packages we prefer `optionals` since it cannot be accidentally passed an array resulting in array-within-array, and it also aligns better when multiple `lib.optionals` are used.